src/efi.rs

```diff
@@ -137,6 +142,7 @@ impl Efi {
         if let Some(mount) = self.mountpoint.borrow_mut().take() {
             Command::new("umount")
                 .arg(&mount)
+                .arg("-l")
```

Comment:
I still can't really think of why this would be needed.

The only reason I can think of is the buffers hadn't been flushed yet, which maybe what we should do instead is run a `sync` after `apply_diff` rather than immediately unmounting and only `umount` after the `sync` is done.

Reply:
Sorry that I should check more about this, see that there is `syncfs` after `apply_diff` (see https://github.com/coreos/bootupd/blob/main/src/filetree.rs#L402), but failed with 
```
umount: /boot/efi: target is busy.
[DEBUG bootupd::efi] Unmounting
error: Failed adopt and update: unmount after adopt: Failed to unmount "/boot/efi": Child ["umount" "/boot/efi"] exited: exit status: 32
```

If append `-l` after `umount`, no error, so I think we should keep it, WDYT?